Docs 3287 (#2179)
MultiSafepay/docs: content/my-account/onboarding/websites.md

1. Sign in to your MultiSafepay dashboard. 
2. Go to **Settings** > **Website settings**.
3. Click either:  
    
    <details id="quick-add-website">

>     <summary>Quick add website</summary>

Reviewer: ```suggestion
    <summary>Quick add website</summary>
    <br>
```

4. From the **Category** list, select what type of products or services your website sells.
5. In the **Description** field, enter the website name.  
    **Note:** If relevant, this is displayed on MultiSafepay payment pages and the customer’s bank statement.
6. In the **Base URL** field, add the website’s URL. This must be the URL where you receive payments.

> 7. If you want to receive [status updates](/about-payments/multisafepay-statuses/) via webhook, in the **Notification URL** field, enter a URL for us to send them to.

Reviewer: ```suggestion
7. If you want to receive [status updates](/payment-statuses/) via webhook, in the **Notification URL** field, enter a URL for us to send them to.
```

8. Click **Save**.

    </details> 
    Or, for more advanced configuration:

>     <details id="website-wizard" >

Reviewer: ```suggestion
    <details id="website-wizard">
```

>     <summary>Website wizard</summary>

Reviewer: ```suggestion
    <summary>Website wizard</summary>
    <br>
```

8. In the **Save template as** field, enter a name for this template.  

Your account is now linked to your website. 

>     </details >

Reviewer: ```suggestion
    </details>
```
